apps/cowswap-frontend/src/modules/trade/hooks/setupTradeState/useSetupTradeStateFromUrl.ts

```diff
@@ -1,12 +1,20 @@
 import { useSetAtom } from 'jotai'
 import { useMemo, useRef } from 'react'
 
+import { useSearchToken } from '@cowprotocol/tokens'
+
 import { useLocation, useParams } from 'react-router'
 
 import { tradeStateFromUrlAtom } from '../../state/tradeStateFromUrlAtom'
 import { TradeRawState } from '../../types/TradeRawState'
 import { useTradeState } from '../useTradeState'
 
+const getMaybeChainId = (chainId: string | undefined | null): number | null => {
```

Comment:
Nitpick:
Does it need the `maybe` prefix?
The type already indicates it can return `null`, and it's a pretty common pattern to return a falsy value throughout the code if I'm not mistaken.